stylesheets/_topics.scss

```diff
@@ -410,6 +410,12 @@ ul.topic-resources.topic-resources-list{
   max-width: 85%;
 }
 
+// Style for custom microservices topic page
+.main-promo.microservices {
+  background-image: url(http://static.jboss.org/_root/images/rhd/topic/RHDev_topics_primopromo_microservicesicon_20sept2016.png);
+  background-size: cover;
+}
+
```

Comment:
Shouldn't this go with its counterparts in https://github.com/KamiQuasi/developers.redhat.com/blob/36b9d4f5576116646a24cbf0a6586c7dd6f22b94/stylesheets/_topics.scss#L83-L108